__tests__/app.test.js

```diff
@@ -42,6 +42,40 @@ describe('GET /api', () => {
     })
 })
 
+describe('GET /api/articles/:article_id', () => {
+    test('status 200: should return an article object containing the 8 properties the article in the database has', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.article_id).toEqual(1)
+            expect(body.title).toEqual("Living in the shadow of a great man")
+            expect(body.topic).toEqual("mitch")
+            expect(body.author).toEqual("butter_bridge")
+            expect(body.body).toEqual("I find this existence challenging")
+            expect(body.created_at).toBe("2020-07-09T20:11:00.000Z")
+            expect(body.votes).toBe(100)
+            expect(body.article_img_url).toEqual("https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700")
+        })
```

Comment:
Would a more flexible matcher be more appropriate for this test? Like using expect.objectContaining or .toMatchObject 🤔